package.json

```diff
@@ -269,6 +269,7 @@
 		"babel-plugin-dynamic-import-node": "^2.3.0",
 		"babel-plugin-transform-react-remove-prop-types": "^0.4.24",
 		"browserslist": "^4.12.0",
+		"cache-loader": "^4.1.0",
```

Comment:
Nitpick: given that this is only used inside `./client`, could you move it to `./client/package.json`?

Reply:
Didn't know this - good catch! Will do.